tests/sentry/auth/providers/github/test_client.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from sentry.testutils import TestCase
+from sentry.utils.compat.mock import patch
+
+from sentry.auth.providers.github.client import GitHubClient
+
+
+class MockReq:
+    def __init__(self, status_code=200, content=None):
+        self.status_code = status_code
+        self.content = content
+
+
+class MockHttp:
```

Comment:
You could likely use `responses` to do capture the requests made and not need to mock json.loads as well. The [docs for responses](https://github.com/getsentry/responses#dynamic-responses) have some examples, and the ones that use `responses.calls[0].request.params` are likely relevant to what you're doing.